SunEngine.Commons/Utils/ServiceCollectionEx.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace SunEngine.Commons.Utils
+{
+    public static class ServiceCollectionEx
```

Comment:
What does Ex mean? I would also be careful with utils classes, it's good to have libraries as utils (work with files for example, or for some calculation), as for higher levels abstractions - usually it worth to create service responsible for logic. Utils tends to grow large because of vague responsibilities in them.

Reply:
Ex - Extension

Reply:
I would suggest using Extension instead of Ex then. It's considered the bad practice to have abbreviations in code - it could be hard to understand what it's mean (Ex- could be Exception), it could lead to misunderstandings, it's also leaves feeling that coder just lazy(and it's not true for sure).